package.json

```diff
@@ -1,15 +1,6 @@
 {
   "name": "visual",
-  "scripts": {
-    "postinstall": "pbiviz update 1.6.0",
-    "pbiviz": "pbiviz",
-    "package": "pbiviz package",
-    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src)/**/*.ts\"",
-    "start": "pbiviz start"
```

Comment:
Please return the scripts back as it's used for building this visuals and CI.
Please update API version in postinstall command.

Reply:
done